manager/manager.go

```diff
@@ -953,11 +970,21 @@ func (m *manager) createContainerLocked(containerName string, watchSource watche
 		if err != nil {
 			klog.Warningf("Error getting perf_event cgroup path: %q", err)
 		} else {
-			cont.perfCollector, err = m.perfManager.GetCollector(perfCgroupPath)
+			cont.collectors[perfMetricsProviderKey], err = m.metricsManagers[perfMetricsProviderKey].GetCollector(perfCgroupPath)
 			if err != nil {
 				klog.Infof("perf_event metrics will not be available for container %s: %s", cont.info.Name, err)
 			}
 		}
+
+		cpuCgroupPath, err := handler.GetCgroupPath("cpu")
+		if err != nil {
+			klog.Warningf("Error getting cpu cgroup path: %q", err)
+		} else {
+			cont.collectors[wssMetricsProviderKey], err = m.metricsManagers[wssMetricsProviderKey].GetCollector(cpuCgroupPath)
```

Comment:
instead of using the manager model, can we instead put this in the container collector code?  I don't think there is any initialization or anything like that that needs to be done.  This seems more along the lines of process scheduler metrics: https://github.com/google/cadvisor/blob/master/container/libcontainer/handler.go#L72

Reply:
I moved getting of wss into libcontainer/handler.go. Do you think that is an good idea to move the rest of the code from wss package into libcontainer?

Reply:
Yes, that would be good.